logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/SendCallingMessage.kt

```diff
@@ -0,0 +1,16 @@
+package com.wire.kalium.logic.feature.call
+
+import com.wire.kalium.logic.data.conversation.ClientId
+import com.wire.kalium.logic.data.id.ConversationId
+import com.wire.kalium.logic.data.message.Message
+import com.wire.kalium.logic.data.message.MessageContent
+import com.wire.kalium.logic.data.user.UserId
+import java.util.Date
+import java.util.UUID
+
+suspend fun CallManager.sendCallingMessage(conversationId: ConversationId, userId: UserId, clientId: ClientId, data: String) {
+    val messageContent =  MessageContent.Calling(data)
+    val date = Date().toString()
+    val message =  Message(UUID.randomUUID().toString(), messageContent, conversationId, date, userId, clientId, Message.Status.SENT)
+    messageSender.getRecipientsAndAttemptSend(conversationId, message)
+}
```

Comment:
I'm wondering about testing. It's something @alexandreferris and I haven't solved before, so I am just venting and I wouldn't request changes to this PR.

But for the future. I guess we can do this to all the handlers, and test them as small units:

```kotlin
class SendHandlerImpl(private val messageSender: MessageSender, private val calling: Calling): SendHandler{ 
    override fun onSend(
        context: Pointer?,
        conversationId: String,
        userIdSelf: String,
        clientIdSelf: String,
        userIdDestination: String?,
        clientIdDestination: String?,
        data: Pointer?,
        length: Size_t,
        isTransient: Boolean,
        arg: Pointer?
    ): Int {
        TODO("Handle the callback here")
    }
} 
```

Otherwise, `CallManager` will just grow out of proportion. It has far too many responsibilities. We could delegate them to these small guys.
`sendCallingMessage` as an extension function works, but it's hard to test and forces us to leave all the dependencies public inside `CallManager`.